starter/script.js

```diff
@@ -1 +1,73 @@
-// put code here!
+function startGame() {
+var name = prompt('What is your NAME?');
+'hello '+ name + '!';
+var Class = prompt('choose WARRIOR or ROGUE');
+  'you have chosen' + Class + '!';
+  if (Class == 'ROGUE'){
+    banditResponse = prompt('a bandit camp lay ahead do you FIGHT or SNEAK?');
+  }
+  if (banditResponse == 'FIGHT'){
+    death();
+  }
+    else if (banditResponse == 'SNEAK'){
+      riverChoice = prompt('a river is ahead do you SWIM or take the BRIDGE a mile away?');
+  }
+  if (riverChoice == 'BRIDGE'){
+    death();
+  }
```

Comment:
Would be mindful of indentation in your conditionals as it is easy to lose track of nested control flows.